.gitignore

```diff
@@ -22,5 +22,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-/.idea/
 /.idea
+/.idea/workspace.xml
```

Comment:
you can actually add this to your global gitignore! But fine to leave it in here for now. :)